cnd/src/btsieve/ethereum.rs

```diff
@@ -202,8 +201,7 @@ where
         match block_generator.async_resume().await {
             GeneratorState::Yielded(block) => {
                 let block_hash = block
-                    .hash
-                    .ok_or_else(|| anyhow::anyhow!("block without hash"))?;
+                    .hash;   // .ok_or_else(|| anyhow::anyhow!("block without hash"))?;
```

Comment:
Please remove the comment and inline the variable.

Reply:
done